packages/website/src/index.html

```diff
@@ -56,8 +56,7 @@
     <!--placeholder1-->
     <div id="root"></div>
     <!--/placeholder1-->
-    <!-- FIXME we should use async instead of defer -->
-    <script src="/main.tsx" defer type="module"></script>
+    <script src="/src/main.tsx" async type="module"></script>
```

Comment:
Please revert this.

Reply:
This would make vercel fail though!

Reply:
sorry, I was talking about the FIXME comment and "defer".